lib/floe/workflow/intrinsic_function/parser.rb

```diff
@@ -64,6 +64,7 @@ class Parser < Parslet::Parser
           :states_base64_encode,   "States.Base64Encode",
           :states_base64_decode,   "States.Base64Decode",
           :states_hash,            "States.Hash",
+          :states_json_merge,      "States.JsonMerge",
```

Comment:
This order is not alphabetical - it currently matches the order as listed in the stateslanguage.net spec.  The same goes for the order of the methods within the transformer.